nifi-nar-bundles/nifi-salesforce-bundle/nifi-salesforce-processors/src/main/java/org/apache/nifi/processors/salesforce/QuerySalesforceObject.java

```diff
@@ -194,9 +193,8 @@ public class QuerySalesforceObject extends AbstractProcessor {
     static final PropertyDescriptor INITIAL_AGE_FILTER = new PropertyDescriptor.Builder()
             .name("initial-age-filter")
             .displayName("Initial Age Filter")
-            .description("When 'Age Field' is set the value of this property will serve as a filter when this processor runs the first time."
-                    + " Only records that are older than this value be retrieved."
-            )
+            .description("Unless this parameter is specified as an initial start time, the time window's left edge is open" +
+                    " on the first run, and all records are fetched until the current runtime.")
```

Comment:
As mentioned in the other property, the `left edge` terminology could be confusing.

What do you think of the following?

```suggestion
            .description("This property specifies the start time that the processor applies when running the first query.")
```

Reply:
I think it's simple and clear, thanks!